src/Gui/WorkbenchManipulatorPython.cpp

```diff
@@ -153,6 +153,10 @@ void WorkbenchManipulatorPython::tryModifyMenuBar(const Py::Dict& dict, MenuItem
         std::string command = static_cast<std::string>(Py::String(dict.getItem(remove)));
         if (auto par = menuBar->findParentOf(command)) {
             if (MenuItem* item = par->findItem(command)) {
+                if (item == menuBar) {
+                    // Can't remove the menubar itself - Coverity issue 512853
+                    return;
```

Comment:
If we come here, then we most likely have an error somewhere, should we log it with FC_DEBUG()?